### PR TITLE
Create testing for notifyMoreshift including mock supervisor API

### PR DESCRIPTION
**Pull request by apackin**
#### What's this PR do?
1: Create mock Supervisor API with comprehensive test data
2: Tests functionality of NotifyMoreShifts
#### Where should the reviewer start?
test/notifyMoreShifts.js
#### How should this be manually tested?
npm test
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-186
#### Questions:
Some of my tests for NotifyMoreShifts depends on other imported files. Should we exclusively test those separately?

### CODE REVIEW

File: test/sampleData.js [hunk 1]
```diff
@@ -372,6 +372,236 @@ var wiwShiftObject =
...
+      positions: [],
+      locations: [ 1003762, 1003765 ],
+      position_rates: [],
+      position_quality: [],
+      sort: { '1003762': 0, '1003765': 0 } 
+    }
+  ],
 };
 
 // These are used in the return for wiwRequestObject
```
Comment (mbogor) [inline test/sampleData.js]: sooo extensive, love it. my only hesitation is having all the supervisor/trainers emails available publicly, other than that, good to merge.

Comment (apackin) [inline test/sampleData.js]: Great point!

File: email_templates/composeNotifyMoreShifts.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ function composeSupTiles (shift, shiftToSup) {
     '<img src=' + sup.imgURL + ' alt="'+ fullName+ '" height="150" width="150">' +
     '</div>';
   }
-
+
```
Comment (tongxiang) [inline email_templates/composeNotifyMoreShifts.js]: Let's take this out to keep this file's commit history clean.

Comment (apackin) [inline email_templates/composeNotifyMoreShifts.js]: Can you show me how to do that?

File: jobs/scheduling/WiWSupervisorsApi.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+var WhenIWork = require('wheniwork-unofficial');
```
Comment (tongxiang) [inline jobs/scheduling/WiWSupervisorsApi.js]: I don't like this is placed within the `jobs` directory. 

These API initialization files should be placed in a top-level directory. 

We currently have 4-5 files which each initialize a separate WhenIWork object. I want to take some time, perhaps in another PR, to refactor this and minimize the number of files which are initializing an API object. 

For now, I think it'd be best for us to move this API object to a top-level directory. (It's not a job.)

**Comment by tongxiang** [comment]
The "creates correct mandrill messages" tests are failing when I pull down. 

otherwise, everything else looks good.

**Pull request merged**